src/renderer/screens/exchange/Sell/SelectAccountAndCurrency.js

```diff
@@ -108,7 +108,7 @@ const SelectAccountAndCurrency = ({
   const dispatch = useDispatch();
 
   const openAddAccounts = useCallback(() => {
-    dispatch(openModal("MODAL_ADD_ACCOUNTS", { currency }));
+    dispatch(openModal("MODAL_ADD_ACCOUNTS", { currency, skipChooseCurrencyStep: true }));
```

Comment:
correct me if i'm wrong but to me we don't need to introduce a new flag, the simple fact there is a currency in prop is enough to know we should "skip"

Reply:
@gre As far as I can see ( I checked the code and tested it) there is no provision for skipping a step in the current code, what we do have is a means to provide the currency to prefill, as we do on the manager, it merely opens the add account modal with the currency pre-filled. What am I missing 🤓 ?


https://user-images.githubusercontent.com/4631227/130210300-6a3e3726-e708-43bb-89d5-37942e5c6fd2.mov

Reply:
@gre @juan-cortes In terms of the code. Thanks for your feedback. I take into account all of them and they should be all fixed now. You can review again.
@juan-cortes For the feature, what I understand is that we provide the currency to prefill in the "choosecurrency" step if we know the currency, but to simplify the flow to add an account, we skip this "choosecurrency" step if we already know the currency. this can save one click in terms of the user experience.
Correct me if I make a mistake. Thanks